WikiWatershed/model-my-watershed
# Update Microsite CSV

The microsite CSV has been regenerated with precipitation levels which are whole numbers of centimeters.  The five levels of precipitation (in centimeters) are: 1, 3, 5, 8, 21.

**To Test**
   * Update the CSV by typing `./scripts/manage.sh collectstatic`
   * Use the microsite.  All of the precipitation levels should be whole-numbers of centimeters, and everything should appear more-or-less as before.

Connects WikiWatershed/tr-55#44
